src/hybrid_a_star_planner/path_planner/reeds_shepp_path.py

```
@@ -0,0 +1,92 @@
+        + steer_angle_change_cost
+    )
+
+    return current_node.cost + cost_increment
+
+
+def reeds_shepp_expansion(
+    current_node: HybridAStarNode, goal_node: HybridAStarNode, map_params: MapParameters
+) -> Optional[HybridAStarNode]:
+    """
+    尝试从当前节点到目标节点找到一条 Reeds-Shepp 路径作为启发式或最终路径。
+    返回找到的路径节点，如果没有找到或有碰撞则返回 None。
```

Ruff (0.14.8)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)